### PR TITLE
Changes due to ATCamera updates from ts_xml 4.3.0

### PR DESCRIPTION
**Pull request by menanteau**
Changes due to ATCamera updates from ts_xml 4.3.0

### CODE REVIEW

File: bin/telemetry_sim_ATHS [hunk 1]
```diff
@@ -216,8 +231,11 @@ if __name__ == "__main__":
 
         # Send the startIntegration for image: k
         LOGGER.info("Sending simulated starIntegration")
```
Comment (mareuter) [inline bin/telemetry_sim_ATHS]: Should be _startIntegration_

Comment (menanteau) [reply bin/telemetry_sim_ATHS]: Oops. Thank for catching that.

**Review by mareuter — COMMENTED** [review]
Looks good.

**Comment by mareuter** [comment]
I think everything matches the mapping table now. There is one item that's missing from the YAML file that's in the mapping table: _TEMP_SET_. If this it to be implemented later, the PR can be merged.

**Comment by menanteau** [comment]
> I think everything matches the mapping table now. There is one item that's missing from the YAML file that's in the mapping table: _TEMP_SET_. If this it to be implemented later, the PR can be merged.

Thanks for catching this one. I've added `TEMP_SET` to the yaml file as `ATCamera_vacuum_tempCCDSetPoint`

**Pull request merged**